lib/db/job.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE job(
+    id int primary key,
```

Comment:
please assign comments for each column-you can assign comments or describe each column of a table in PostgreSQL